domains/ehsanshopbd.json

```diff
@@ -0,0 +1,6 @@
+"owner": {
+    "username": "ehsanshopbd",
+    "email": "ehsanshopbd@gmail.com"
+  "records": {
+    "CNAME": "ehsanshopbd.github.io"
+  }
```

Comment:
Invalid, this is broken JSON. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.